android/app/src/main/java/org/mozilla/magnet/permissions/Check.java

```diff
@@ -0,0 +1,55 @@
+package org.mozilla.magnet.permissions;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.pm.PackageManager;
+import android.support.v4.content.ContextCompat;
+
+/**
+ * Created by wilsonpage on 09/11/2016.
+ */
+abstract class Check {
```

Comment:
+1 to all the comments from @samgiles about specifying the visibility of the class and methods, otherwise the visibility is just for the package.

Also I would rename the class, `Check` is too generic. I know we have it under the package permissions, but I like verbosity in class names.